ecr/Makefile

```diff
@@ -0,0 +1,31 @@
+all: init plan apply aws build tag push
+
+#Terraform
+
+init: 
+	terraform init
+
+plan: init
+	terraform plan 
+
+apply: plan 
+	terraform apply -auto-approve
```

Comment:
I don't like the auto-approve.. we should always check the plan before applying, so:
- remove the `plan` linked with apply
- update to `terraform apply .terraform-plan` 

that way once you run `make plan` you get the results planned to change and if you do a `make apply` you won't need to type `yes`